b2share/modules/records/errors.py

```diff
@@ -36,3 +41,16 @@ class AlteredRecordError(B2ShareRecordsError):
     """Raise when a record update changes what is considered
        immutable record data."""
 
+
+def register_error_handlers(app):
+    @app.errorhandler(ValidationError)
+    def handle_ValidationError(err):
```

Comment:
I would rather keep to PEP8 naming style when possible (https://www.python.org/dev/peps/pep-0008/#function-names), i.e: `handle_validation_error`